core/src/main/java/tech/pegasys/web3signer/core/config/Config.java

```diff
@@ -37,6 +38,8 @@ public interface Config {
 
   Path getKeyConfigPath();
 
+  YAMLMapper getKeyConfigYamlMapper();
```

Comment:
This shouldn't be part of the config. To me, this class is meant to represent the configuration that is passed into the application. The YamlMapper it the output of that.